spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/ApiKeyFetcher.java

```diff
@@ -53,7 +53,7 @@ public ApiKeyFetcher(RestTemplate restTemplate, UrlResourcesProvider urlResource
     }
 
     public List<ApiKey> requestLatestApiKeyListFromAdmin() {
-        log.debug("Requesting latest API key list from CoatRack admin.");
+        log.debug("Requesting latest API key list from " + urlResourcesProvider.getApiKeyListRequestUrl());
```

Comment:
This is from another branch or?
It seems that here 3 different issues were joined:
1. The property to allow switching off sending emails
2. The issue about making the Gateway logs better readable for the end users
3. The template to our private profiles
Shouldn't we put those code changes on different MRs?

Reply:
To clarify: 
- this has nothing to do with (2.), that is a separate issue. 
- this is built on top of (3.), therefore it is still a draft. After merging master, there will only be the changes from (1.).

Reply:
This was discussed and it is now solved.